r/vignettes/developing.Rmd

```diff
@@ -202,11 +226,12 @@ To enable optional features including: S3 support, an alternative memory allocat
 Other flags that may be useful:
 
 * `-DBoost_SOURCE=BUNDLED` and `-DThrift_SOURCE=bundled`, for example, or any other dependency `*_SOURCE`, if you have a system version of a C++ dependency that doesn't work correctly with Arrow. This tells the build to compile its own version of the dependency from source.
```

Comment:
```suggestion
* `-DBoost_SOURCE=BUNDLED` and `-DThrift_SOURCE=BUNDLED`, for example, or any other dependency `*_SOURCE`, if you have a system version of a C++ dependency that doesn't work correctly with Arrow. This tells the build to compile its own version of the dependency from source.
```

I believe these are both not case sensitive (though we should probably check). This is something I noted when looking at this page a few days ago but didn't have a change to come and fix it separately.